### PR TITLE
vci 0.4.2

### PR DESCRIPTION
**Pull request by acidjazz**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  vci
    * GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)
```

**Comment by acidjazz** [comment]
> ```
>   vci
>     * GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)
> ```

Thanks! I'll work on that.  I'll comment this PR once I hopefully meet that requirement

File: Formula/v/vci.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  end
+
+  depends_on "go" => :build
+
+  deny_network_access! [:postinstall, :test]
+
+  def install
+    with_env(
+      "VC_VERSION" => version.to_s,
+      "GO_LDFLAGS" => "-s -w -X main.updaterEnabled=vulncheck-oss/cli",
```
Comment (chenrui333) [inline Formula/v/vci.rb:22]: what is the ref for `updaterEnabled`?

Comment (acidjazz) [reply Formula/v/vci.rb:22]: This is a reference/placeholder for a future feature where the cli will notify the user if it's out of date in some certain situations

Comment (chenrui333) [reply Formula/v/vci.rb:22]: we dont allow self-update though

**Pull request closed without merging**